packages/scene-composer/src/components/WebGLCanvasManager.tsx

```diff
@@ -54,6 +56,14 @@ export const WebGLCanvasManager: React.FC = () => {
     }
   }, [environmentPreset]);
 
+  useEffect(() => {
```

Comment:
it is best to avoid using `useEffect` where possible, please read: https://react.dev/learn/you-might-not-need-an-effect

For example in this case, it would actually be better in your render function to just write:

`const groundMeshOrder = matterportModelId ? 1 : undefined`

Reply:
seems more like a `useMemo` case, and also simple enough to just add to the rendering part

Reply:
I don't think `useMemo` is appropriate here - caching won't save us anything on a simple computation like this unless i'm missing something